pkg/metrics/record.go

```diff
@@ -139,31 +139,33 @@ func RecordApplyDuration(ctx context.Context, status, commit string, startTime t
 }
 
 // RecordResourceFight produces measurements for the ResourceFights view.
-func RecordResourceFight(ctx context.Context, _ string, _ schema.GroupVersionKind) {
-	//tagCtx, _ := tag.New(ctx,
-	//tag.Upsert(KeyName, GetResourceLabels()),
-	//tag.Upsert(KeyOperation, operation),
-	//tag.Upsert(KeyType, gvk.Kind),
-	//)
+func RecordResourceFight(ctx context.Context, operation, kind string) {
+	reconcilerName := os.Getenv(reconcilermanager.ReconcilerNameKey)
+	tagCtx, _ := tag.New(ctx,
+		tag.Upsert(KeyName, reconcilerName),
```

Comment:
I don't think these can/should be added if they're not registered on the metric View. Otherwise there will be errors when sending tags that aren't in the metric scheme.

Same goes for all the other tags you uncommented.

Reply:
/cc @tiffanny29631

Reply:
Should we add these tags to the metric views?

Reply:
I think that should be a different PR, if we do. And we need to discuss the cardinality effects, which is why they were removed.

Reply:
Make sense. Commented back.